src/components/DeliverableView/StatusBar.tsx

```diff
@@ -1,12 +1,32 @@
-import { Box } from '@mui/material';
+import { Box, useTheme } from '@mui/material';
 import { ViewProps } from './types';
+import DeliverableStatusBadge from 'src/components/DeliverableView/DeliverableStatusBadge';
+import strings from 'src/strings';
 
 const StatusBar = (props: ViewProps): JSX.Element => {
+  const { deliverable, isAcceleratorConsole } = props;
+  const theme = useTheme();
+
   return (
     <Box display='flex' flexDirection='column'>
-      <div>
-        StatusBar coming soon {props.isAcceleratorConsole}! Current status {props.deliverable.status}
-      </div>
+      {deliverable.status === 'Rejected' && (
+        <Box
+          border={`1px solid ${theme.palette.TwClrBaseGray100}`}
+          borderRadius='8px'
+          marginBottom='16px'
+          padding='16px'
+        >
+          <DeliverableStatusBadge status={deliverable.status} />
+          <strong>{strings.REASON}</strong> {deliverable.reason}
+        </Box>
+      )}
+
+      <Box border={`1px solid ${theme.palette.TwClrBaseGray100}`} borderRadius='8px' marginBottom='16px' padding='16px'>
```

Comment:
this entire box should only be visible to accelerator console (internal comments are not shared with external users/participants).

Reply:
Good catch, thanks! That makes sense that they wouldn't be shared externally, will update.